### PR TITLE
(DOCSP-24168) AtlasCLI version 1.1.5

### PR DESCRIPTION
**Pull request by zach-carr**
[Ticket](https://jira.mongodb.org/browse/DOCSP-24168)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-24168/atlas-cli-changelog/)

To be backported to stable

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,82 @@
...
    :class: singlecol
 
+.. _atlas-cli-1.1.5:
+
+{+atlas-cli+} 1.1.5
+-------------------
+
+Released 2 August 2022
+
+- Fixes a bug where telemetry tracking events were not propagated when
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:22]: Suggestion: link to telemetry page
```suggestion
- Fixes a bug where :ref:`telemetry <telemetry>` tracking events were not propagated when 
```

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -12,6 +12,82 @@
...
+- Fixes a bug where some options were not validated on command 
+  :ref:`atlas-backups-restores` job start.
+
+- Improves documentation and adds examples to 
+  :ref:`atlas-privateEndpoints-aws-describe`.
+
+- Improves onboarding for users with more than 500 organizations.
+
+- Improves the message when an operation completes in a free 
+  cluster, directing users to an upgrade command.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:37]: I think this is what was meant by the original sentence but wasn't sure, feel free to change if needed
```suggestion
  cluster. The message now directs users to an upgrade command.
```

File: source/atlas-cli-changelog.txt [hunk 3]
```diff
@@ -12,6 +12,82 @@
...
+
+- Updates the help message for roles in :ref:`atlas-dbusers-create`
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:39]:
```suggestion
- Updates the help message for roles in the :ref:`atlas-dbusers-create` 
```

File: source/atlas-cli-changelog.txt [hunk 4]
```diff
@@ -12,6 +12,82 @@
...
+  - :ref:`atlas-backups-schedule-delete` deletes all backup schedules 
+    for a given cluster.
+
+  - :ref:`atlas-backups-schedule-update` updates the backup schedule of 
+    a cluster.
+
+  - :ref:`atlas-backups-schedule-describe` gets the backup schedule of 
+    a cluster.
+
+  - :ref:`atlas-backups-restores-describe`
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:70]: Did you mean to have a description for this, or to remove it?

File: source/atlas-cli-changelog.txt [hunk 5]
```diff
@@ -12,6 +12,82 @@
...
+
+  - :ref:`atlas-backups-exports-jobs-create` creates a new export job 
+    in a given AWS S3 bucket.
+
+  - :ref:`atlas-backups-exports-jobs-describe`
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:75]: Needs description

File: source/atlas-cli-changelog.txt [hunk 6]
```diff
@@ -12,6 +12,82 @@
...
+
+  - :ref:`atlas-backups-exports-jobs-list`
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:77]: Needs description

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
A couple of changes, mostly just the ones that need a description

File: source/atlas-cli-changelog.txt (hunk 2 above)
Comment (zach-carr) [reply source/atlas-cli-changelog.txt:37]: That's my understanding too

File: source/atlas-cli-changelog.txt (hunk 4 above)
Comment (zach-carr) [reply source/atlas-cli-changelog.txt:70]: No, I just didn't add one if it wasn't in the notes. Adding now :)

**Review by sarahsimpers — APPROVED** [review]
LGTM!

**Pull request merged**